data/exotic-synergies.ts

```diff
@@ -0,0 +1,179 @@
+import { getAllDefs } from '@d2api/manifest-node';
+import { DamageType } from 'bungie-api-ts/destiny2/interfaces.js';
+import { getComposedRegex } from '../src/helpers.js';
+import { PlugCategoryHashes } from './generated-enums.js';
+
+const inventoryItems = getAllDefs('InventoryItem');
+
+export const synergies = {
+  arc: {
+    super: [
+      getSingleSuperNameAndHash('fists of havoc'),
+      getSingleSuperNameAndHash('thundercrash'),
+      getSingleSuperNameAndHash('stormtrance'),
+      getSingleSuperNameAndHash('chaos reach'),
+      getSingleSuperNameAndHash('arc staff', 'whirlwind guard'),
+      getSingleSuperNameAndHash('gathering storm'),
```

Comment:
might be able to automate this by filtering on itemCategoryHash===50 && talentGrid.hudDamageType === DamageType.Arc